lib/types/rules.d.ts

```diff
@@ -422,6 +422,18 @@ export interface ESLintRules extends Linter.RulesRecord {
 		[
 			Partial<{
 				exceptMethods: string[];
+				/**
+				 * @default true
+				 */
+				enforceForClassFields: boolean;
+				/**
+				 * @default false
+				 */
+				ignoreOverrideMethods: boolean;
+				ignoreClassesWithImplements:
+					| "all"
+					| "public-fields"
+					| undefined;
```

Comment:
Do we need to specify `undefined` since it is already under `Partial`?
```suggestion
				ignoreClassesWithImplements:
					| "all"
					| "public-fields";
```

Reply:
Got it!